lib/models/message.dart

```diff
@@ -0,0 +1,34 @@
+class Message {
+  final MessageType type;
+  final MessageReceiverType receiverType;
+  final String? teamId;
+  final String? value;
+
+  Message(
+      {required this.type,
+      required this.receiverType,
+      this.teamId,
+      this.value});
+
+  Message.fromJson(Map<String, String> json)
+      : type = MessageType.values.byName(json['type']!),
+        receiverType = MessageReceiverType.values.byName(json['receiverType']!),
+        teamId = json['teamId'],
```

Comment:
it will return null if such key is not defined, right?

Reply:
Yes